src/frontend/src/functions/studentprofileinfo.js

```diff
@@ -28,13 +29,20 @@ export async function updateStudentProfileInfo(phone_number,
        },
        data : data
     };
-    
-    axios(config)
-    .then(function (response) {
-       console.log(JSON.stringify(response.data));
-       getCurrentStudentInfo();
-    })
-    .catch(function (error) {
-       console.log(error);
-    });
+
+    await axios(config)
+        .then(function (response) {
+            console.log(JSON.stringify(response.data));
+            setUserInfoJson(response.data);
+            notification.success({
+                message: 'Profile Edited!',
+            });
+        })
+        .catch(function (error) {
+            console.log(error);
+            notification.error({
+                message: 'Profile Edition Failed',
+                description: 'Please try again'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MarkHershey/SUTDHousingPortal/pullRequest?prid=7298734)
```suggestion
                description: "Please try again"
```